config/examples/basic_pipeline.yaml

```diff
@@ -3,6 +3,10 @@ type: config
 log_level: DEBUG
 cpu_affinity: [0,1,2,3]
 
+trackers_dump:
```

Comment:
I would rename this slightly.   Call `trackers_dump:` just `trackers:` and then `path:` -> `dump_path:`, `enabled:` -> `enable_crash_dump`   That way if we want to add additional parameters, we have a logical part of the configuration to add them.